schemas/2015-05-01-preview/Microsoft.Storage.json

```diff
@@ -0,0 +1,91 @@
+{
+  "id": "https://schema.management.azure.com/schemas/2015-05-01-preview/Microsoft.Storage.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.Storage",
+  "description": "Microsoft Storage Resource Types",
+  "resourceDefinitions": {
+    "storageAccounts": {
+      "type": "object",
+      "properties": {
+        "apiVersion": {
+          "type": "string",
+          "enum": [
+            "2015-05-01-preview"
+          ]
+        },
+        "location": {
+          "type": "string",
+          "description": "Resource location"
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the storage account within the specified resource group. Storage account names must be between 3 and 24 characters in length and use numbers and lower-case letters only.  "
+        },
+        "properties": {
+          "oneOf": [
+            {
+              "$ref": "#/definitions/StorageAccountPropertiesCreateParameters"
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/common/definitions.json#/definitions/expression"
+            }
+          ]
+        },
+        "tags": {
+          "oneOf": [
+            {
+              "type": "object",
+              "additionalProperties": {
+                "type": "string"
+              },
+              "properties": {}
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/common/definitions.json#/definitions/expression"
+            }
+          ],
+          "description": "Resource tags"
+        },
+        "type": {
+          "type": "string",
+          "enum": [
+            "Microsoft.Storage/storageAccounts"
+          ]
+        }
+      },
+      "required": [
+        "apiVersion",
+        "location",
+        "name",
+        "properties",
```

Comment:
This `properties` is actually not required, according to @blueww

Reply:
Can we add a post-processing step to mark it as non-required?

Reply:
We'll try this solution